storops/unity/resource/lun.py

```diff
@@ -370,7 +370,7 @@ def _is_move_session_supported(self, dest):
             return False
         return True
 
-    def migrate(self, dest, **kwargs):
+    def migrate(self, dest, provision=None, **kwargs):
```

Comment:
Change the name of `provision` to `dest_provision`. And leave a comment about the possible value ('thin', 'thick', 'compressed') of it.

Reply:
will add (None, 'thin', 'thick', 'compressed') possible values in comment.
Move this part to cinder code as suggested.